### DISCUSSION TITLE
量算工具自定义单位

### DISCUSSION THREAD

**Discussion by X717zq** [zouyaoji/vue-cesium#520] [Q&A] [answered]
请问量算的时候如何自定义单位转换，目前需要转换成亩，内置的参数貌似无法设置

**Comment by zouyaoji** [comment]
确实，我看了一下，需要在

https://github.com/zouyaoji/vue-cesium/blob/dev/packages/shared/src/MeasureUnits.ts#L14

添加亩的单位，或者干脆把相关转换方法areaToString暴露出来，你自己转换：

https://github.com/zouyaoji/vue-cesium/blob/dev/packages/composables/use-drawing/use-drawing-polyline.ts#L209

**Reply by MangMax** [reply]
感觉直接自己传递方法转换会比较灵活，因为单位也需要自定义

**Reply by zouyaoji** [reply, marked as answer]
OK，下个版本可以自定义了。

![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/84c04a7d-8ca4-44bd-a1f2-c8c42edc2396)

**Comment by X717zq** [comment]
好的，感谢！
